crates/rspack_core/Cargo.toml

```diff
@@ -15,6 +15,7 @@ dashmap = "5.0.0"
 derivative = "2.2.0"
 dyn-clone = "1.0.10"
 futures = "0.3.21"
+glob = "0.3.1"
 glob-match = "0.2.1"
```

Comment:
Would you mind explaining why introduce `glob` since we already have `glob-match` ?

Reply:
glob-match has no custom configuration, although glob's configuration is not that much